.github/workflows/gpu_test.yaml

```diff
@@ -1,4 +1,4 @@
-name: Multi-GPU Recipe Tests
```

Comment:
Why the rename here?

Reply:
It's no longer just a recipe test, right? Now it's recipe + unit test